x11-packages/qt6-qtscxml/build.sh

```diff
@@ -0,0 +1,43 @@
+TERMUX_PKG_HOMEPAGE=https://www.qt.io/
+TERMUX_PKG_DESCRIPTION="Qt6 SCXML Library"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="6.8.0"
+TERMUX_PKG_SRCURL="https://download.qt.io/official_releases/qt/${TERMUX_PKG_VERSION%.*}/${TERMUX_PKG_VERSION}/submodules/qtscxml-everywhere-src-${TERMUX_PKG_VERSION}.tar.xz"
+TERMUX_PKG_SHA256=bba9aa4ef441b01b2d391486ddccfb09efa378c8a24e6d13e983fc019837b43b
+TERMUX_PKG_DEPENDS="libc++, qt6-qtbase, qt6-qttools, libxkbcommon, zlib"
+TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_NO_STATICSPLIT=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DCMAKE_SYSTEM_NAME=Linux
+-DCMAKE_INSTALL_PREFIX=${TERMUX_PREFIX}/opt/qt6/cross
+-DINSTALL_PUBLICBINDIR=${TERMUX_PREFIX}/opt/qt6/cross/bin
+-DQT_HOST_PATH=${TERMUX_PREFIX}/opt/qt6/cross
+"
+
+termux_step_host_build() {
+	termux_setup_cmake
+	termux_setup_ninja
+
+	cmake \
+		-G Ninja \
+		-S ${TERMUX_PKG_SRCDIR} \
+		-DCMAKE_BUILD_TYPE=MinSizeRel \
+		-DCMAKE_DISABLE_FIND_PACKAGE_Clang=ON \
+		-DCMAKE_INSTALL_PREFIX=${TERMUX_PREFIX}/opt/qt6/cross \
+		-DINSTALL_PUBLICBINDIR=${TERMUX_PREFIX}/opt/qt6/cross/bin
+	ninja \
+		-j ${TERMUX_PKG_MAKE_PROCESSES} \
+		install
+}
+
+termux_step_make_install() {
+	cmake \
+		--install "${TERMUX_PKG_BUILDDIR}" \
+		--prefix "${TERMUX_PREFIX}" \
+		--verbose
```

Comment:
This `--verbose` was probably left over from testing.

Reply:
Hmm. I think I just copied that from `qt6-qtsvg/build.sh`, and it appears to be present in most `qt6-*` builds. Should we remove them all?

Reply:
Eh we can remove it from the other `qt6-*` packages in a follow up PR/commit, it's not really in the scope of this PR.

My reasoning is mostly to keep the logs concise for the package builds.